examples/calls/list_call_logs.bal

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/io;
+import ballerina/os;
+import ballerinax/twilio;
+
+// Account configurations
+configurable string accountSID = os:getEnv("ACCOUNT_SID");
+configurable string authToken = os:getEnv("AUTH_TOKEN");
+
+// This sample demonstrates a scenario where Twilio connector is used to list all call logs.
+public function main() returns error? {
+
+    twilio:ConnectionConfig twilioConfig = {
+        auth: {
+            username: accountSID,
+            password: authToken
+        }
+    };
+
+    twilio:Client twilio = check new (twilioConfig);
+
+    twilio:ListCallResponse responce = check twilio->listCall();
+
+    twilio:Call[]? calls = responce.calls;
+    if calls is twilio:Call[] {
+        calls.forEach(function(twilio:Call call) {
+            io:println(`Date&Time: ${call?.date_created}  From: ${call?.'from}  To: ${call?.to}  Status: ${call?.status} SID: ${call?.sid}`);
```

Comment:
Why don't we just print the `call` value in here?

Reply:
Yes sure. Addressed in [474f18f](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/474f18fdb6516b8ff29c450126cde2c618369527)